app/components/primer/alpha/action_list/heading.html.erb

```diff
@@ -1,5 +1,5 @@
-<%= render(Primer::BaseComponent.new(**@system_arguments)) do %>
-  <%= render(Primer::BaseComponent.new(tag: @tag, classes: "ActionList-sectionDivider-title", id: @list_id)) do %>
+<%= render(Primer::BaseComponent.new(tag: :div, **@system_arguments)) do %>
+  <%= render(Primer::BaseComponent.new(tag: @tag, classes: "ActionList-sectionDivider-title", id: @list_id, role: :heading, aria: { level: @heading_level })) do %>
```

Comment:
Does this really need a `role="heading"` if its a heading tag?

Reply:
Agreed if its `h1-h6` we shouldn't need the role or aria-level

Reply:
I think this must have just been us misinterpreting the a11y feedback 🥴